### DISCUSSION TITLE
New Inverted Coaster types track pieces

### DISCUSSION THREAD

**Discussion by zViper25** [OpenRCT2/OpenRCT2#23138] [Feature requests] [answered]
With all of the recent love given to the Corkscrew and Wooden Roller Coaster Types, are new track pieces for the Inverted and Compact Inverted Coaster types in the cards?

I'd love to see the new medium loop, large corkscrews, Diagonal banked/unbanked sloped turns,  and especially steep-to-level slope pieces added to specifically the B&M Inverted Coaster type considering how massive some of the rides can be. I'm not sure if these are specifically more difficult to implement than standard sit-down trains but I was surprised that I couldn't find anyone else mention it.

**Comment by TheAtomTwister** [comment]
Making coasters like Great Bear and Banshee would be cool.  Pairing the large half loop with a large Zero-G roll makes an Immelmann loop for Great Bear.  A pair would make the pretzel loop that Banshee has.  Great Bear also has a Zero-G roll like also does Batman: the Ride.  This might just have to wait in line though, making these track pieces from what I've learned is not exactly easy.  I'm still excited for it.  I would like to know the steps to doing this in case one day I want to take on the job myself.

**Comment by tbarford** [comment, marked as answer]
https://github.com/OpenRCT2/OpenRCT2/pull/25964
